includes/emails/class-wp-job-manager-email-admin-updated-job.php

```diff
@@ -78,7 +78,8 @@ public function get_from() {
 	 * @return string|array
 	 */
 	public function get_to() {
-		return get_option( 'admin_email', false );
+		$option = get_option( 'job_manager_email_admin_updated_job', false );
+		return ! $option['email_to'] ? get_option( 'admin_email', false ) : $option['email_to'];
```

Comment:
You can use the elvis operator to simplify these:
```suggestion
		return get_option( 'job_manager_email_admin_updated_job', false ) ?: get_option( 'admin_email', false );
```

You need to be a bit careful as the check will fail for all 'falsy' values but I think this is ok in this case.

Reply:
Good suggestion!  It looks like our PHPCS rules don't allow this though? What are your thoughts?  
```
 81 | ERROR | Using short ternaries is not allowed (WordPress.PHP.DisallowShortTernary.Found)
```

Reply:
Ah yes forgot about this one. In general we could add a comment to ignore the rule but I don't think we should it in this case. Can we at least switch the order to avoid the negation?
```
$option['email_to'] ?  $option['email_to'] : get_option( 'admin_email', false );
```

Reply:
Done in 6e6bc581ed0f4972abb1f6eb7d4ab5881049302f